frontend/src/shared/components/Toast/constants/toast.constants.ts

```diff
@@ -0,0 +1,16 @@
+import errorIcon from "@assets/icons/error-icon.svg";
+import infoIcon from "@assets/icons/info-icon.svg";
+import successIcon from "@assets/icons/successful-icon.svg";
+import warningIcon from "@assets/icons/warning-icon.svg";
+
+export const DEFAULT_TYPE = "info";
+export const DEFAULT_POSITION = "bottom-center";
+export const DEFAULT_DURATION_MS = 4000;
+export const TOAST_LIMIT = 3;
+export const TOAST_ICONS = {
```

Comment:
```suggestion
export const TOAST_DEFAULT_TYPE = "info";
export const TOAST_DEFAULT_POSITION = "bottom-center";
export const TOAST_DEFAULT_DURATION = 4000;
export const TOAST_LIMIT_COUNT = 3;
export const TOAST_TYPE_ICONS = {
```
`DEFAULT_TYPE`은 import 할 때 사용하는 곳에서 너무 추상적인 표현인 것 같아서 `TOAST`를 붙여도 좋을 것 같네요!

Reply:
저는 개인적으로 `success`, `error`를 더 많이 본 것 같은데 세오는 기본 타입을 `info`로 생각한 이유도 궁금하네요~

Reply:
상수에 TOAST 프리픽스를 붙이는 것을 고려했지만, 이미 파일명(`toast.constants.ts`)과 사용처에서 toast 맥락이 명확하기 때문에 불필요하다고 판단했습니다! 

icon 폴더 안의 파일들에 굳이 `-icon` 접미사를 붙이지 않는 것과 같은 맥락이라고 생각해요.

기본 타입을 `info`로 선택한 이유는 `success`나 `error`는 명확한 성공/실패 상황에서 의도적으로 사용하는 것이 좋다고 생각했기 때문입니다. 반면 `info`는 단순한 정보 전달 목적으로 가장 범용적으로 사용할 수 있어서 기본값에 적합하다고 판단했습니다!

Reply:
써밋이 말씀해주신 DEFAULT_TYPE 네이밍 충돌 가능성에 대해서는 충분히 공감합니다.
다만 현재 toast.constants.ts 내 상수들은 toast 도메인 내부에서만 사용되며, modal이나 alert와 같은 다른 도메인과 공유되는 구조가 아니어서 실제로 겹칠 상황은 거의 없다고 생각했습니다.

이 경우 TOAST_ 접두어가 중복 정보를 제공할 수 있다고 판단했습니다.

다만, 말씀 주신 것처럼 여러 도메인 상수가 한 곳에 모여서 import되는 구조가 생길 경우에는 접두어가 유용할 수 있을 것 같은데 혹시 써밋이 생각하시는 여러 도메인 상수가 한 곳에 모이는 경우가 있으실까요?🤔